Scuti/Scripts/UI/Store/TopBarView.cs

```diff
@@ -20,27 +20,42 @@ public class TopBarView : View {
     public List<BannerWidget> additionalBanners;
     private bool isAdditionalBanners;
 
+    
     private void Awake()
+    {
+        // Only the default banner is enrolled.        
+        Banner.onCreateBanners += CreateBanners;
+      
+    }
+
+    private void CreateBanners(int amount)
```

Comment:
amount and amountBanners is confusing.  

amount could be offerCount
amountBanners could be bannerCount 

Or something along those lines to show one is the data and one is the visuals

Reply:
Ok I will change them.